### DISCUSSION TITLE
[Question]get rid of round corners

### DISCUSSION THREAD

**Discussion by retsyo** [flet-dev/flet#1814] [Q&A]
### Question

currently, the corners of the controls/widgets are round. Any way to set the corners are 90 degrees? Thanks

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
It depends on the control.

**Comment by ndonkoHenri** [comment]
The corners of which control/widget do you mean @retsyo ?

**Comment by retsyo** [comment]
so does it mean there is no global setting to turn on/off round corner?

**Reply by ndonkoHenri** [reply]
There isn't... 
Provide more context on your issue :)

**Comment by retsyo** [comment]
Currently, AlertDialog, TextButton, ElevatedButton, Column, Dropdown, TextField, Tooltip. Thanks
